packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -1268,3 +1637,58 @@ function synthesizeBlockDeviceMappings(construct: Construct, blockDevices: Block
     };
   });
 }
+
+/**
+ * Options for applying CloudFormation init to an instance or instance group
+ */
+export interface ApplyCloudFormationInitOptions {
+  /**
+   * ConfigSet to activate
+   *
+   * @default ['default']
```

Comment:
What does  `default` mean here?

Reply:
The configset with the literal name `"default"`.